src/homeWork1/task2/task2.js

```diff
@@ -0,0 +1,11 @@
+import fullFileWriter from './fullFileWriter/fullFileWriter'
+import byLineFileWriter from './byLineFileWriter/byLineFileWriter.js'
+
+const csvFilePath = __dirname + '/csv/nodejs-hw1-ex1.csv'
+const txtFilePath = __dirname + '/txt/file.txt'
```

Comment:
👍 it is better to use `path.join(__dirname, 'csv/nodejs-hw1-ex1.csv')`

Reply:
fixed